src/components/GoalsComponents/GoalSublistPage/GoalSublistPage.tsx

```diff
@@ -33,7 +41,7 @@ export const GoalSublist = () => {
   const callResetHistory = useSetRecoilState(resetGoalsHistory);
   const setShowAddGoalOptions = useSetRecoilState(displayAddGoalOptions);
 
-  const [tapCount, setTapCount] = useState(defaultTap);
+  const [tapCount, setTapCount] = useState<{ open: string | number, click: number }>(defaultTap);
```

Comment:
Please do not engage yourself in refactoring. this creates conflicts in coming PR(s). Refactoring is done only when there is no **feature** in progress.